src/olympia/devhub/templates/devhub/versions/list.html

```diff
@@ -118,9 +118,15 @@
                   <p><a href="#" class="review-history-loadmore" data-div="#{{ version.id }}-review-history">{{ _('Load older...') }}</a></p>
               </div>
               <div class="review-entry-empty hidden">
-                  <p><strong class="action">$action_label</strong> {{ _('by') }}
-                  <em class="user_name">$user_name</em> <time class="timeago" datetime="$date">$date</time></p>
-                  <pre>$comments</pre>
+                  <div>
+                    <p><strong class="action">$action_label</strong> {{ _('by') }}
+                    <em class="user_name">$user_name</em> <time class="timeago" datetime="$date">$date</time></p>
+                    <pre>$comments</pre>
+                  </div>
+                  <div class="review-entry-attachment hidden">
```

Comment:
Doubling back on my comment in the javascript file. You should be able to wrap the attachment div in a jinja condition to only render if there is an attachment. Then you can pass all the properties expecting them to exist, I think. Unless there is some reason why the attachment would  only be available client side (which I doubt)

Reply:
> Unless there is some reason why the attachment would only be available client side

This statement seems to be true. Yeah I get it now.